crates/sema/src/lib.rs

```diff
@@ -25,6 +25,7 @@ mod ast_lowering;
 mod ast_passes;
 
 mod parse;
+use crate::hir::{Arena, Hir};
```

Comment:
should maintain style of the other fns, using hir:: as a module instead of importing

Reply:
changed in https://github.com/paradigmxyz/solar/pull/210/commits/9c654ccb33690e5664274e324dadf80e779aa369